config-templates/pgsql_tables.sql

```diff
@@ -21,7 +21,13 @@ CREATE TABLE identityprovidersmap (
     name character varying(255) NOT NULL
 );
 
+ALTER TABLE identityprovidersmap
+    ADD CONSTRAINT identityprovidersmap_pk PRIMARY KEY (entityid);
+
```

Comment:
Better move PK constraint to `CREATE TABLE` statement (see also `CREATE TABLE statistics`):
```
CREATE TABLE identityprovidersmap (
    entityid character varying(255) NOT NULL,
    name character varying(255) NOT NULL,
    PRIMARY KEY (entityid)
);
```